README.rst

```diff
@@ -52,18 +52,23 @@ During SCT environment setup, you will be asked to configure your AWS CLI tool.
 QA private keys from secure S3 Bucket during automated setup.
 The keys will be needed to connect to the Scylla clusters under test via SSH.
 
-Use following values when running ``aws configure``
- * ``region`` -> `us-east-1`
- * ``Default output format`` -> `None`
- * ``aws_access_key_id``, ``aws_secret_access_key``
+Ask Shlomi Livne to create a user and access key for AWS) and then configure AWS::
```

Comment:
This is open source repo, should we mention employee in Scylla as part of the procedure?
@roydahan WUT?